dipy/reconst/recspeed.pyx

```diff
@@ -203,16 +223,15 @@ def search_descending(cnp.ndarray[cnp.float_t, ndim=1, mode='c'] a,
     i : int
         The greatest index such that ``all(a[:i] >= relative_threshold *
```

Comment:
The first thing I wondered when I read the doc string was "what happens if no such index i exist?, how am I going to catch that case?", this line answers the question but it was a little cryptic for me at the beginning. To help the inexperienced reader (like me), can we add something like "If no such index 'i' exists, the length of a is returned instead." in the description of the function? (Is that statement correct?).